t/sql/mozart.sql

```diff
@@ -0,0 +1,188 @@
+-- Automatically generated, do not edit.
```

Comment:
The added test data is generated with `script/dump-entities-sql.pl` (which I ran from astro). We should probably change the script so that it prints the script arguments on top in a comment...but it's just "$entity_type $mbid1 $mbid2 ..."